src/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -613,12 +613,16 @@ public static IEnumerable<object[]> Groups_CustomCulture_TestData()
         [MemberData(nameof(Groups_CustomCulture_TestData))]
         public void Groups(string pattern, string input, RegexOptions options, CultureInfo cultureInfo, string[] expectedGroups)
         {
-            CultureInfo originalCulture = null;
+            CultureInfo originalCulture = CultureInfo.CurrentCulture;
             try
             {
+                // In invariant culture, the unicode char matches differ from expected values provided.
+                if (originalCulture == CultureInfo.InvariantCulture)
+                {
+                    CultureInfo.CurrentCulture = new CultureInfo("en-US");
+                }
                 if (cultureInfo != null)
                 {
-                    originalCulture = CultureInfo.CurrentCulture;
                     CultureInfo.CurrentCulture = cultureInfo;
                 }
```

Comment:
hmm but if the dev is setting cultureinfo to invariant, then that's a conscious decision on what to test for.. Changing that behavior is not what we want..